src/Controller/PageAdminController.php

```diff
@@ -273,6 +274,14 @@ public function composeContainerShowAction(Request $request): Response
         \assert($blockManager instanceof BlockServiceManagerInterface);
         $blockServices = $blockManager->getServicesByContext('sonata_page_bundle', false);
 
+        foreach ($blockServices as $code => $blockService) {
+            if ($blockService instanceof EditableBlockService) {
+                continue;
+            }
+
+            unset($blockServices[$code]);
+        }
```

Comment:
We're doing this logic at least twice.

Would it make sens to add/modify a method in BlockServiceManager then ?
If most of the time/always we need to work on EditableBlockService only in PageBundle, it might make sens.

WDYT ?

Reply:
BlockServiceManager is from block-bundle, how could we handle this change?

Reply:
I was wondering if we should introduce an EditableBlockServiceManager in BlockBundle then.
Or introduce getEditableService and getEditableServiceByContext...

But this solution is currently the only one we have